src/xcode/ENA/ENA/Source/Views/ENSetting/EuActionTableViewCell.swift

```diff
@@ -0,0 +1,46 @@
+//
+// Corona-Warn-App
+//
+// SAP SE and all other contributors
+// copyright owners license this file to you under the Apache
+// License, Version 2.0 (the "License"); you may not use this
+// file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+//
+
+import UIKit
+
+// TODO (KGA): Rethink ActionCell inheritance, not implemented for the moment
```

Comment:
when will this be addressed?

Reply:
I do not have this code base locally. It was removed (refactored) by me a while ago. I don't know why this is shown in GitHub.